plugin/js/showpass-custom.js

```diff
@@ -124,37 +124,91 @@
 				showpass.tickets.calendarWidget(id, params);
 			});
 
-			const embeddedCalendarExists = document.getElementById('showpass-calendar-widget');
-			if (embeddedCalendarExists) {
-				let script = document.createElement("script");
-				script.type = "text/javascript";
-				script.src = 'https://showpass.com/static/dist/sdk.js';
-
-				let useBeta = $('#option_use_showpass_beta').val();
-				let useDemo = $('#option_use_showpass_demo').val();
-				if (useBeta) {
-					script.src = 'https://beta.showpass.com/static/dist/sdk.js';
-				} else if (useDemo) {
-					script.src = 'https://demo.showpass.com/static/dist/sdk.js';
-				}
+			function initializeShowpassEmbeddedWidgets() {
+				const embeddedCalendarWidget = document.getElementById('showpass-calendar-widget');
```

Comment:
I'm confused by this line. Why are we assuming this is the id of where we will mount? Are we creating this element in the plugin?

Reply:
Yes we are creating the element in the plugin

Reply:
Where is that happening? I dont see `createElement` anywhere

Reply:
this happens here https://github.com/showpass/showpass-wordpress-plugin/blob/0438dd81e743eecfcb43ecb98b6f3b78a0d6422f/plugin/showpass-wordpress-plugin-shortcode.php#L991-L995